CarRental/CarRental.Web/Views/Home/_AvailableOptionsForYearTo.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@ViewData.Eval("Year")
```

Comment:
jw.
niepotrzebny kolejny plik na partial
zamiast dwóch partiali po prostu można użyć jeden (i z nazwą bez końcówki From ani To)